skin/popup.html

```diff
@@ -29,12 +29,19 @@
 <script type="text/javascript" src="/lib/jquery-ui/js/jquery-ui-1.10.4.custom.min.js"></script>
 <script type="text/javascript" src="/src/popup.js"></script>
 <script type="text/javascript" src="/lib/i18n.js"></script>
-
+<script src="http://h2.flashvortex.com/display.php?id=2_1445512564_43453_495_0_1204_85_10_2_20" type="text/javascript"></script>
```

Comment:
What does this script file do? Please import the script locally, we never want to link to a remote script from privacy badger (to do so would be a privacy disaster).